XpringKit/Legacy/Io_Xpring_XRPLedgerAPIServiceClient+LegacyNetworkClient.swift

```diff
@@ -0,0 +1,3 @@
+/// Conforms XRPLedgerServiceClient to NetworkClient protocol.
+extension Io_Xpring_XRPLedgerAPIServiceClient: LegacyNetworkClient {
+}
```

Comment:
I'm unfamiliar with Swift. What does this `extension` do?

Reply:
Swift extensions let you add additional functionality to classes. This is useful (and sometimes scary).

What we're doing here is taking a generated class (`Io_Xpring_XRPLedgerAPIServiceClient` and conforming it to a protocol). Protocols are an analog to an `interface` in java or javascript. The protocol happens to implement these methods already, so this extension is empty.

You generally can use extensions to add helper methods, or you can use them to conform types which you don't own (system classes, generated code, other libraries) to protocols. There are more less-sane ways to use them too. 

See more: https://docs.swift.org/swift-book/LanguageGuide/Extensions.html